packages/codegen.go/clientFactory.ts

```diff
@@ -3,10 +3,10 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import * as go from '../gocodemodel/gocodemodel';
+import * as go from '../codemodel.go/gocodemodel.js';
```

Comment:
Is it possible to import with `@azure-tools/codemodel.go`? Maybe we need to release a first version for it.

Reply:
I explicitly chose this design as I want to avoid having another dependency. Releasing `codemodel.go` as a node package would mean that updating the code generator (autorest and/or typespec) means updating/releasing `codemodel.go` then the respective code generators.